server/svix-server/src/core/webhook_http_client.rs

```diff
@@ -764,4 +777,61 @@ mod tests {
             "127.0.0.1:8000".as_bytes()
         );
     }
+
+    #[tokio::test]
+    async fn test_tls_verification_disable() {
```

Comment:
If we are already testing this dangerous flag, maybe also have a test to test the happy path? So if it's false, that we actually fail connecting to it?

Reply:
That is one of the assertions in this test case already.

Reply:
Ah, missed that, awesome.